### ISSUE TITLE
CD Stuck to 8 versions ago for app01

### ISSUE THREAD

**Issue by palladius** [palladius/clouddeploy-platinum-path#17]
My app has been upgraded this morning from 2.19 to 2.20. 
However, the dev service still points to v2.14. I've shot in the face Service and pod, I've restored the service via cloud deploy but nothing,, still 2.14

Let me show you.

```$ cat apps/app01/VERSION 
2.20
```
```
[DEBUG] DEBUG has been enabled! Please change to DEBUG=FALSE in your .env.sh to remove this. Some impotant fields:
[DEBUG] PROJECT_ID:        'cicd-platinum-test002'
[DEBUG] ACCOUNT:           'palladiusbonton@gmail.com'
[DEBUG] GITHUB_REPO_OWNER: 'palladius'
[DEBUG] GCLOUD_REGION:     'europe-west1'
[DEBUG] GKE_REGION:        'europe-west1'
Getting status of my 8 apps them all WOW:
== app01: python web app ==
HTTP_RESPONSE: [DEV] 200 for http://34.78.131.190:8080/ ''
HTTP_RESPONSE: [STAG] 200 for http://34.76.157.232:8080/        ''
HTTP_RESPONSE: [CANA] 200 for http://34.140.176.245:8080/       ''
HTTP_RESPONSE: [PROD] 200 for http://35.187.20.43:8080/ ''
```

The freshest is dev, look : http://34.78.131.190:8080/ 

=> ```
App01 (🐍) v2.14
Hello world from Skaffold in python! 
[...]
app01 (🐍) v2.14 TARGET=None```

(Yes I've tried reloading and curling - same result)

**Comment by palladius** [comment]
Note, this does NOT happen with Ruby (which is at latest version 1.35). Just with python. Weird.

**Comment by palladius** [comment]
Alex is currently investigating

**Comment by palladius** [comment]
Fix was this: 4731ecea28798739ab34963c09e157ec8a850468

**Comment by aablsk** [comment]
Fixed and deployed
